modules/claims_api/spec/lib/claims_api/error/error_handler.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
```

Comment:
Just noticed this, but the file name needs to end with `_spec.rb`.
So `error_handler_spec.rb`.